src/reducers/account/index.ts

```diff
@@ -1,10 +1,14 @@
 import { combineReducers } from 'redux'
 import { did } from 'src/reducers/account/did'
+import { claims } from 'src/reducers/account/claims'
+import { Map } from 'immutable'
 
 export interface AccountState {
   readonly did: string
+  readonly claims: Map<string, any>
```

Comment:
I will probably add stronger typings here in a follow up PR, like in https://github.com/jolocom/smartwallet-app/pull/1024